README-DEVELOPMENT.md

```diff
@@ -15,21 +15,20 @@ This page lists contains rudimentary instructions for building the development e
    
    `docker-compose up --build`
 
-1. All subsequent commands should be run from the `server` directory
+3. All subsequent commands should be run from the `server` directory
```

Comment:
I agree the steps here are confusing - you actually don't have to cd to `server` directory because it is running the command inside the container, so I removed that.
dpcreator-server does give you a hint that migrations need to be done. 
```
dpcreator-server-1        | You have 47 unapplied migration(s). Your project may not work properly until you apply the migrations for app(s): account, admin, analysis, auth, authtoken, banner_messages, contenttypes, dataset, dataverses, release_schemas, sessions, sites, socialaccount, terms_of_access, user.
dpcreator-server-1        | 
dpcreator-server-1        | Run 'python manage.py migrate' to apply them.
```
I agree it would be nice to have a reminder at the end of the build.